test/android/core_internal/javatests/src/org/xwalk/core/internal/xwview/test/ExtensionEchoInternalTest.java

```diff
@@ -33,7 +33,7 @@ public void testAsync() throws Throwable {
     public void testSync() throws Throwable {
         ExtensionEchoInternal echo = new ExtensionEchoInternal();
 
-        loadAssetFile("echoSync.html");
+        loadAssetFile("echoSync_java.html");
```

Comment:
Please rename it as `echo_sync_java.html`, it is your not fault, I'd like to make it right to follow chromium coding style.